api/src/main/java/org/apache/iceberg/view/ViewVersion.java

```diff
@@ -65,5 +65,8 @@ public interface ViewVersion {
    *
    * @return the string operation which produced the view version
    */
-  String operation();
+  @Value.Derived
+  default String operation() {
+    return summary().get("operation");
+  }
 }
```

Comment:
Nice, Derived is way cleaner. We don't need any of the special  parsing logic for operation and derived also prevents manually setting the operation field, so there is no ambiguity in which is the real "operation"